examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb

```diff
@@ -372,6 +372,7 @@
     "    dask_cluster_kwargs={\n",
     "        \"n_workers\": 3,\n",
     "        \"worker_size\": \"xlarge\",\n",
+    "        \"scheduler_size\": \"medium\",\n",
```

Comment:
this isn't necessary (no customers on current releases should be facing this), but it avoids this bug: https://github.com/saturncloud/dask-saturn/pull/54

I think it's worth adding here to be sure people who get these notebooks but `dask-saturn` 0.2.2 don't hit the error described in that PR.

Reply:
I did test this live on `app.daily` (should have mentioned that before) and confirmed that it avoided the `dask-saturn` bug mentioned above